IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DoorTypeTurnstile/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of Automatic gate used in railway. The property set can be used by the predefined type TURNSTILE of IfcDoor
```

Comment:
```suggestion
Properties of common to turnstyles or automatic gate used to control the flow of people or vehicles. The property set is applied to IfcDoor instances of predefined type TURNSTILE.
```

Definition is railway specific and therefore should be generalised to cover other possibilities e.g. stadiums, airports etc.